src/qililab/analog/annealing_program.py

```diff
@@ -41,11 +42,15 @@ class AnnealingProgram:
         annealing_program (list[dict[str, dict[str, float]]]): dictionary with the annealing program with the above structure.
     """
 
-    def __init__(self, platform: Any, annealing_program: list[dict[str, dict[str, float]]]):
+    def __init__(
+        self,
+        flux_to_bus_topology: list[Runcard.FluxControlTopology],
+        annealing_program: list[dict[str, dict[str, float]]],
+    ):
         """Init method"""
-        self._platform = platform
```

Comment:
we really need docstrings here for the flux topology.

Reply:
And also merge latest from main as there was a bug related to flux_topology and runcard serialization that Guille did a hotfix for.